recipes/capnproto/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "0.5.3" %}
+
+package:
+  name: capnproto
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
+  url: https://github.com/sandstorm-io/capnproto/archive/v{{ version }}.tar.gz
+  md5: 909bd13ad6b8bc840ac78ab8f5bcb0a4
+
+build:
+  number: 0
+  script_env:
+    - MAKEFLAGS
```

Comment:
Removed this from the recipe since it was useful only for local frequent rebuilds on Linux, which is working fine now.